SQL/New_patches/2018-10-01-Add_data_release_permissions.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO permissions (code,description,categoryID) VALUES ('data_release_upload','Data Release: Upload file',2), ('data_release_edit_file_access','Data Release: Grant other users view-file permissions',2);
```

Comment:
Should not use 2 as the categoryID but query the permissions_category table for exact value.
This presume that in every instance, it is ID 2.